packages/core/src/modules/generic-record/GenericRecordModule.ts

```diff
@@ -0,0 +1,77 @@
+import type { Logger } from '../../logger'
```

Comment:
Maybe we can call the module generic-records? All other modules are plural

Reply:
Done